### PR TITLE
nwd&nww

### PR DESCRIPTION
**Pull request by pako2200**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @pako2200

**Comment by ziobron** [comment]
⏱ 3 XP extra for @pako2200 for on-time delivery

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
-    return -1;
+int NWD(int lhs, int rhs)
+{
+    int rest;
+    if (rhs == 0)
+        return lhs;
+    else if (lhs == 0)
+        return rhs;
+    else
+    {
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:11]: If you return variable here, you don't need to use `else if` or `else`

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
-int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+int NWW(int lhs, int rhs)
+{
+    if (rhs == 0)
+        return rhs;
+    else if (lhs == 0)
+        return lhs;
+    else
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:30]: If you return variable here, you don't need to use `else if` or `else`

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**